test/clique_leader_test.go

```diff
@@ -0,0 +1,33 @@
+package test
+
+import (
+	"testing"
+	"math/rand"
+	"log"
+	"io/ioutil"
+	"github.com/krzysztof-turowski/distributed-framework/leader/clique"
+	"github.com/krzysztof-turowski/distributed-framework/lib"
+)
+
+func TestHumblet(t *testing.T) {
+	checkLogOutput()
+
+	for n := 2; n <= 100; n++ {
+		nodes, runner := lib.BuildCompleteGraph(n)
+		clique.RunHumblet(nodes, runner)
+	}
+}
+
+func BenchmarkHumblet(b *testing.B) {
+	b.StopTimer()
```

Comment:
On the second thoughts: maybe it's better to put some timer in Runner e.g.:
- start the timer before "for finish := 0; finish < r.n; {" line in lib_runner.go,
- end the timer after the for loop,
- let runner.GetStats report the time spent,
instead of meddling with the benchmark timer?

Reply:
Done.